app/scripts/lib/app-start.js

```diff
@@ -173,6 +174,37 @@ function (
       };
     },
 
+    _checkForRefresh: function (currentPath) {
+
+      var storage = Storage.factory('sessionStorage', this._window);
+
+      var refreshMetrics = storage.get('last_page_loaded');
+
+      if (refreshMetrics && refreshMetrics.url === currentPath) {
+        if (this._metrics) {
+          this._metrics.logEvent('refresh(' + currentPath + ')');
```

Comment:
1. I don't think we have / support `(` or `)` in our metrics infrastructure. 

2. each view has a `logScreenEvent` (and a screen name) that you might find useful instead of `this._window.location.pathname`